src/Arcus.EventGrid.Proxy.sln

```diff
@@ -1,9 +1,9 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.28307.168
+# Visual Studio Version 16
```

Comment:
Wow, it must have been a long time since this source code was updated 😄 .

Reply:
Jup 😅